app/src/main/java/org/wikipedia/gallery/GalleryActivity.kt

```diff
@@ -123,7 +124,7 @@ class GalleryActivity : BaseActivity(), LinkPreviewDialog.LoadPageCallback, Gall
         galleryAdapter = GalleryItemAdapter(this@GalleryActivity)
         binding.pager.adapter = galleryAdapter
         binding.pager.registerOnPageChangeCallback(pageChangeListener)
-        binding.pager.offscreenPageLimit = 2
+        binding.pager.offscreenPageLimit = ViewPager2.OFFSCREEN_PAGE_LIMIT_DEFAULT
```

Comment:
I'm not sure I agree with this -- it seems to make the user experience worse. The reason for increasing the `offscreenPageLimit` is to make the swiping of images smoother: the next image is _pre-loaded_, so that the swiping shows you the image already loaded.
With this change, the next image is only loaded when you start swiping, which looks much less fluid.